cli/src/validate.rs

```diff
@@ -49,19 +54,25 @@ pub async fn validate(junit_paths: Vec<String>, show_warnings: bool) -> anyhow::
         .collect();
 
     // print results
+    let mut exit = EXIT_SUCCESS;
     let (num_invalid_reports, num_suboptimal_reports) =
         print_validation_errors(&report_validations);
     if num_invalid_reports == 0 {
         print_summary_success(report_validations.len(), num_suboptimal_reports);
-        Ok(EXIT_SUCCESS)
     } else {
         print_summary_failure(
             report_validations.len(),
             num_invalid_reports,
             num_suboptimal_reports,
         );
-        Ok(EXIT_FAILURE)
+        exit = EXIT_FAILURE
     }
```

Comment:
smol nit for a bit of simplification

```suggestion
    let (num_invalid_reports, num_suboptimal_reports) =
        print_validation_errors(&report_validations);
    let exit = if num_invalid_reports == 0 {
        print_summary_success(report_validations.len(), num_suboptimal_reports);
        EXIT_SUCCESS
    } else {
        print_summary_failure(
            report_validations.len(),
            num_invalid_reports,
            num_suboptimal_reports,
        );
        EXIT_FAILURE
    };
```

Reply:
updated, thank you!